crafters/nlp/PDFExtractorSegmenter/__init__.py

```diff
@@ -49,7 +49,8 @@ def craft(self, uri: str, buffer: bytes, *args, **kwargs) -> List[Dict]:
                 page = pdf_reader.getPage(i)
                 text += page.extractText()
 
-            chunks.append(
-                dict(text=text, weight=1.0))
+            if text != "":
```

Comment:
`if text` is equivalent I think in Python and more pythonic

Reply:
True!